tests/appveyor/mariadb.appveyor.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I've got MariaDB installed and running, but some adjustment to the DSN user and password are needed to get those tests to pass